canonical-web-and-design/maas-ui
# Fix KVM checkbox when switching from other OS/Release to Ubuntu 18.04.

## Done
- Fixed KVM checkbox being disabled when switching from any other OS/Release back to Ubuntu 18.04 if 18.04 is the first release in state.

## QA
- Make sure Ubuntu 18.04 is the earliest release available and you have at least 2 releases that aren't Ubuntu (bolla is set up this way).
- Select deployable machines and open the deploy form
- Change selection to Ubuntu 18.04 if not the default, and check that the KVM checkbox is enabled.
- Change the OS and release then change back to Ubuntu
- Check that the checkbox is enabled

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1904
